packages/mantine/src/components/prompt/Prompt.module.css

```diff
@@ -1,45 +1,35 @@
 .root {
-    &[data-variant='success']&[data-size] {
-        .header {
-            background-color: var(--mantine-color-lime-6);
-        }
-    }
-    &[data-variant='warning']&[data-size] {
-        .header {
-            background-color: var(--mantine-color-yellow-5);
-        }
-    }
-    &[data-variant='critical']&[data-size] {
-        .header {
-            background-color: var(--mantine-color-red-6);
-        }
+    --prompt-icon-size: 88px;
+
+    .header {
```

Comment:
Why is this style not in `.header {}` selector? (vs currently `.root .header {}`)

Reply:
Without the scoping inside the `.root` the style is not applied. It's due to the fact that the prompt renders a modal and the modal style is applied after the prompt's

Reply:
ah I see, ok